.github/workflows/cicd.yml

```diff
@@ -101,6 +101,20 @@ jobs:
       code_coverage: true
       gather_dumps: true
 
+  # Run the quick stress tests in GitHub.
+  stress:
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      name: stress
+      # Until there is a dedicated stress test, re-use the perf test.
+      test_command: ebpf_performance.exe
```

Comment:
Aside: I observe that ebpf_performance.exe is not one of the binaries listed in the deploy-ebpf.ps1 script.  Seems like it would be useful to deploy it into a VM for use there, so would be good to add it to that file.